packages/app/src/App.tsx

```diff
@@ -118,7 +119,18 @@ const routes = (
     <Route path="/api-docs" element={<ApiExplorerPage />} />
     <Route
       path="/catalog"
-      element={<CatalogIndexPage initiallySelectedFilter="all" />}
+      element={
+        <CatalogIndexPage
+          filters={
```

Comment:
seems the `CatalogIndexPage` itself also has a `initiallySelectedNamespaces` prop, could have used that too

Reply:
Yes, who ever added that was a smart person, maybe they should use it here! LOL (it was me)